_data-prepper/pipelines/configuration/processors/aggregate.md

```diff
@@ -18,12 +18,139 @@ The following table describes the options you can use to configure the `aggregat
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
 identification_keys | Yes | List | An unordered list by which to group events. Events with the same values as these keys are put into the same group. If an event does not contain one of the `identification_keys`, then the value of that key is considered to be equal to `null`. At least one identification_key is required (for example, `["sourceIp", "destinationIp", "port"]`).
-action | Yes | AggregateAction | The action to be performed for each group. One of the available aggregate actions must be provided or you can create custom aggregate actions. `remove_duplicates` and `put_all` are the available actions. For more information, see [Creating New Aggregate Actions](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/aggregate-processor#creating-new-aggregate-actions).
+action | Yes | AggregateAction | The action to be performed for each group. One of the [available aggregate actions](#available-aggregate-actions) must be provided or you can create custom aggregate actions. `remove_duplicates` and `put_all` are the available actions. For more information, see [Creating New Aggregate Actions](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/aggregate-processor#creating-new-aggregate-actions).
 group_duration | No | String | The amount of time that a group should exist before it is concluded automatically. Supports ISO_8601 notation strings ("PT20.345S", "PT15M", etc.) as well as simple notation for seconds (`"60s"`) and milliseconds (`"1500ms"`). Default value is `180s`.
 
-<!---## Configuration
+## Available aggregate actions
 
-Content will be added to this section.--->
+Use the following aggregate actions to determine how the `aggregate` processes events in each group.
+
+### remove_duplicates 
+
+The `remove_duplicates` action processes the first event for a group immediately and drops any events from the source that duplicate the first. For example, when using `identification_keys: ["sourceIp", "destination_ip"]`:
+
+1. The `remove_duplicates` action processes `{ "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "status": 200 }`, the first event in the source.
+2. Data Prepper drops the `{ "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "bytes": 1000 }` event because the `sourceIp` and `destinationIp` matches the first event in the source.
```

Comment:
```suggestion
2. Data Prepper drops the `{ "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "bytes": 1000 }` event because the `sourceIp` and `destinationIp` match the first event in the source.
```